BlazarService/src/main/java/com/hubspot/blazar/util/ModuleBuildLauncher.java

```diff
@@ -94,6 +94,7 @@ private BuildConfig fetchBuildpack(GitInfo gitInfo) throws IOException, NonRetry
   private BuildConfig mergeConfig(BuildConfig primary, BuildConfig secondary) {
     List<BuildStep> steps = primary.getSteps().isEmpty() ? secondary.getSteps() : primary.getSteps();
     List<BuildStep> before = primary.getBefore().isEmpty() ? secondary.getBefore() : primary.getBefore();
+    Optional<BuildConfig.PostBuildSteps> after = primary.getAfter().isPresent() ? primary.getAfter() : secondary.getAfter();
```

Comment:
We should use the same strategy as `before` because:
a) We don't allow arbitrarily nested configurations
b) Because of A parent configurations should never set `before` or `after` since they define the core of the build.